### PR TITLE
[12.0][fix] stock_secondary_unit, do not overwrite stock.move.line secondary_uom_qty on picking validation

### PR DESCRIPTION
**Pull request by GSLabIt**

### CODE REVIEW

**Comment by GSLabIt** [comment]
@sergio-teruel, @enriquemartin any chance to review this fix? 

This avoid `secondary_unit_qty` to be overwritten on picking validation; I found out on validation `{'product_uom_qty': 0}` is sent to `write` method`stock.move.line`.

**Review by francesco-ooops — APPROVED** [review]
@GSLabIt good fix

File: stock_secondary_unit/models/stock_move.py [hunk 1]
```diff
@@ -104,13 +103,14 @@ def write(self, vals):
...
-                qty = float_round(
-                    move_line_qty / (factor or 1.0),
-                    precision_rounding=move.secondary_uom_id.uom_id.rounding
-                )
-                vals.update({
-                    'secondary_uom_qty': qty,
-                    'secondary_uom_id': move.secondary_uom_id.id,
-                })
+                move_line_qty = vals.get('product_uom_qty', vals.get('qty_done', 0.0))
+                if (move_line_qty > 0):
```
Comment (rousseldenis) [inline stock_secondary_unit/models/stock_move.py:107]: @GSLabIt Please use float_compare

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@GSLabIt And can you rebase ?

**Pull request closed without merging**